config-model/src/test/java/com/yahoo/vespa/model/container/xml/IdentityBuilderTest.java

```diff
@@ -19,16 +23,50 @@
     @Test
     public void identity_config_produced() throws IOException, SAXException {
```

Comment:
Test name should constitute a requirement, e.g. (Require that) identity_config_is_produced_when_deployment_xml_specifies_athenz